### ISSUE TITLE
1.6.0 T4 - Gear Updates - Addition of new "Mixed Special Forces" kit

### ISSUE THREAD

**Issue by AngusBethke** [AngusBethke/saef_mod#5]
In addition to the existing uniforms that are currently using the Mil Bottoms and Jersey tops, I would like to introduce a variety of mixed Special Forces kit to allow for more kit variation in missions. There are already prepared templates in the Google Drive Share (If you need access let me know), it is simply a case of adding the new camo patterns.

The "sa_wehrmacht" kit is already setup in this manor, refer to it if you need any guidance in the style.

**Comment by Mascot19** [comment]
So are we talking camo on the shirts, etc vs the Jersey style previously?

**Comment by AngusBethke** [comment]
Yes, but we're also going to keep the existing versions - it's to allow mission makers more variety in the kit available to use. As an example:

**Woodland Camo**
- Bottoms (Woodland Camo) + Jersey Top
- Bottoms (Woodland Camo) + Combat Shirt Top (Flat Color)
- Bottoms (Flat Color) + Combat Shirt Top (Woodland Camo)
- **Bottoms (Woodland Camo) + Combat Shirt Top (Desert Camo)**
- Bottoms (Woodland Camo) + Combat Shirt Top (Woodland Camo)

**Desert Camo**
- Bottoms (Desert Camo) + Jersey Top
- Bottoms (Desert Camo) + Combat Shirt Top (Flat Color)
- Bottoms (Flat Color) + Combat Shirt Top (DesertCamo)
- **Bottoms (Desert Camo) + Combat Shirt Top (Woodland Camo)**
- Bottoms (Desert Camo) + Combat Shirt Top (Desert Camo)

The pairings in the bolded versions above would be picked based on what we feel like. The Wehrmacht kit has been implemented in this manner:
[Examples](https://imgur.com/a/7wCIW9w)

**Comment by AngusBethke** [comment]
@Mascot19 
**Camo Pairings**
To start  - these will probably rotate based on how they look in game

- Desert (TRPTRN) and Multicam (MCAM) - Base Color Tan (Jersey and Flat)
- Multicam (MCAM) and Desert (TRPTRN) - Base Color Tan (Jersey and Flat)
- Snow (SNW) and Multicam (MCAM) - Base Color White-Grey (Jersey and Flat)
- Flecktarn (FLKTRN) and Woodland-ERDL (ERDL) - Base Color Khaki Green (Jersey and Flat)
- Woodland (WDL) and Woodland-ERDL (ERDL) - Base Color Khaki Green (Jersey and Flat)
- Autumn (AUTM) and Flecktarn (FLKTRN) - Base Color Brown (Jersey and Flat)
- Woodland-ERDL (ERDL) and Desert (TRPTRN) - Base Color Khaki Green (Jersey and Flat)
